components/authz-service/storage/v2/storage.go

```diff
@@ -198,7 +198,7 @@ func DefaultProjects() []Project {
 
 	unassignedProject := Project{
 		ID:       constants_v2.UnassignedProjectID,
-		Name:     "Unassigned Project",
+		Name:     constants_v2.UnassignedProjectID,
```

Comment:
Originally I did not include this--having the change only in the migration--but the integration test failed without it.